cmd/minikube/cmd/delete.go

```diff
@@ -97,7 +98,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 
 	validProfiles, invalidProfiles, err := pkg_config.ListProfiles()
 	profilesToDelete := append(validProfiles, invalidProfiles...)
-
+	glog.Infof("error listing profiless %v", err)
```

Comment:
typo, and should probably check for nil ?

Reply:
thanks done